### PR TITLE
Don't Dismiss Produce Message pane after Send

### PR DESCRIPTION
**Pull request by winnie-chiu**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
In the produce message pane add a new checkbox "Keep the contents", unchecked by default. 
If the checkbox is being checked, leave the key/value content and pane as it is.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by kristi-dev — APPROVED** [review]

**Review by armenuikafka — CHANGES_REQUESTED** [review]
"Keep the contents" is unchecked, but data for key and value are kept with producing new message


https://user-images.githubusercontent.com/104780608/200763938-9afee887-5fdc-4b60-bf75-317e97e5485d.mov

**Comment by Haarolean** [comment]
@winnie-chiu can you fix the issue mentioned in [the comment](https://github.com/provectus/kafka-ui/pull/2854#pullrequestreview-1173446892)?

**Comment by winnie-chiu** [comment]
@Haarolean I feel this task is out of my league. I'm not able to fix the issue mentioned above.

**Comment by Haarolean** [comment]
@winnie-chiu ok no worries, we'll do that :) thank you

File: kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx [hunk 1]
```diff
@@ -110,7 +114,11 @@ const SendMessage: React.FC<{ onSubmit: () => void }> = ({ onSubmit }) => {
       keySerde,
       valueSerde,
     });
-    onSubmit();
+    if (!keepContents) {
+      setValue('key', '');
+      setValue('content', '');
+      onSubmit();
```
[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx:120]: ok

[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx:120]: @Mgrdich this function is not mine, as I understand it, it only closes the sidebar, I will rename it

File: kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx [hunk 2]
```diff
@@ -171,6 +179,18 @@ const SendMessage: React.FC<{ onSubmit: () => void }> = ({ onSubmit }) => {
               )}
             />
           </S.Column>
+          <S.Column>
+            <InputLabel>Keep the contents</InputLabel>
+            <Controller
+              control={control}
+              name="keepContents"
+              render={({ field: { name, onChange, value } }) => (
+                <div>
```
[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx:188]: we do not need div

[the remark this answers is not in the document]
Comment (Mgrdich) [reply kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx]: then we can delete it.

File: kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mgrdich) [reply kafka-ui-react-app/src/components/Topics/Topic/SendMessage/SendMessage.tsx]: yeah i want to understand when we should trigger that close thing , cause from only reading of it , the request the BE is happening  -> so logically `onSubmit` should work, `if (!keepContents) {` `setValues` is what is happening to the ui state of the component.

if it is not onSubmit handler then it should be renamed

**Review by Mgrdich — APPROVED** [review]

**Comment by David-DB88** [comment]
almost done it, remove div and renamed function to closeSidebar()

**Review by Haarolean — CHANGES_REQUESTED** [review]
image
visually looks ugly, pending design solution

**Comment by Haarolean** [comment]
Please use this design solution
https://www.figma.com/file/ZkvysdRUmt9p2PDNU3a9WU/Kafka-UI?node-id=3298-11325

**Comment by David-DB88** [comment]
ok

**Review by Haarolean — APPROVED** [review]

**Pull request merged**